src/main.c

```diff
@@ -71,7 +73,17 @@ static char* g_outputFilename = NULL;		/* in case -p/-i flag we get a filename p
 static int   g_sudokuCount = 1;				/* in case of -n we can the numbers of sudoku that should end up in the PDf (-p) */
 static bool  g_outIsPDF;
 static DIFFICULTY g_level = D_EASY;
-static WINDOW *grid, *infobox, *status;
+static WINDOW *grid, *infobox, *status, *resume;
+static bool is_board_solved = false;		/* if the board is solved, we do not save the state */
```

Comment:
I would propose to remove this variable and not check if the user solved/not ask him if he wants to save.
I would like to just have an option "lets save". And the user can do this when he wants but don't add the overhead of tracking this and asking him.